hammr/commands/quota/quota.py

```diff
@@ -75,6 +61,43 @@ def do_list(self, args):
         except Exception as e:
             return handle_uforge_exception(e)
 
+    @staticmethod
+    def get_quota_nb(quota):
+        return " (" + str(quota.nb) + ")" if quota.limit <= 0 \
```

Comment:
Yes, I agree it's less readable because of the length limit of 120 chars else the ternary condition is better than the simple if...else statement. Anyway, I'll replace my ternaries functions on the next commit.